ui/yarn.lock

```diff
@@ -1650,7 +1650,7 @@
   dependencies:
     copy-webpack-plugin "^5.1.2"
 
-"@neutrinojs/dev-server@9.5.0":
+"@neutrinojs/dev-server@9.5.0", "@neutrinojs/dev-server@^9.5.0":
```

Comment:
The lock file isn't capturing all of the package updates. If I run `yarn` locally after checking out this branch, I see a bunch of changes that need to be captured in this pr.

Reply:
Updated yarn lock.